.travis.yml

```diff
@@ -30,6 +30,7 @@ install:
 script:
 - export gitVersion="$(git describe --match 'v[0-9]*' --dirty='.m' --always)"
 - export TRAVIS_TAG="containerd-${gitVersion#v}"
+- if git rev-parse $TRAVIS_TAG > /dev/null 2>&1 ; then echo "Tag exists, skip the build" ; unset TRAVIS_TAG ; exit 0 ; fi
```

Comment:
is it possible to check the releases page instead maybe? that way we can re-release by deleting a release, i'm not sure if we can delete tag from github easily. (the only way I know is to force push the tag removal w/ git)

Reply:
I guess that deleting the release in github you also delete the tag, bear in mind the tag is added to this repo

Reply:
no, github releases are associated with tags, but deleting a release does not delete a tag.

Reply:
yeah, you are right, you have to select the tag button and delete it manually. 

The current script only release if a tag exists, deleting the release and the tag manually is easy, both are in the same web page

Reply:
sgtm. previously you could only create or delete tags via git protocol rather than the site. I knew they added create but not delete :+1: